spec/models/user_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+RSpec.describe User do
+
+  describe '#validation' do
+    context 'can save' do
+      it "is valid with a name, email, password, password_confirmation" do
+        user = build(:user)
+        expect(user).to be_valid
+      end
+    end
```

Comment:
「パスワードが8文字以上32文字以下であれば登録できること」のテストも追加しましょうー！
文字数制限のバリデーションを設定した際は、閾値でcan/can't両方をテストするのがおすすめです

Reply:
Rspec__postで実施しています。

Reply:
Fackerでパスワードが8文字以上なので、8文字以上32文字以下はできていると思います。

Reply:
8文字はできてますが、32文字でも登録できるかがこのテストだと明確でないので、そこも合わせて記述してあげるとよいです！

こういう書き方もある！という一例です↓

```suggestion
      it "パスワード（password）         8文字以上かつ32文字以下で半角英数字両方を含んでいれば登録できること" do
        user = build(:user, password: "abcd1234" * 4 , password_confirmation: "abcd1234" * 4)
        expect(user).to be_valid
      end
```

Reply:
追加しました！
わかりやすく疑問が晴れました
ありがとうございます！